xcode/libmoai/libmoai.xcodeproj/project.pbxproj

```diff
@@ -11778,7 +11778,7 @@
 				STRIP_INSTALLED_PRODUCT = YES;
 				TARGETED_DEVICE_FAMILY = "1,2";
 				VALIDATE_PRODUCT = YES;
-				WARNING_CFLAGS = "-Wall";
+				WARNING_CFLAGS = "-w";
```

Comment:
The most significant change was here, I'm omitting warnings on Moai.
They were mostly deprecation warnings, but I'm also ok if we want to leave them showing.